pyproject.toml

```diff
@@ -11,7 +11,7 @@ dependencies = [
     "matplotlib>=3.6",
     "traitlets>=5.0.5",
     "bqplot>=0.12.37",
-    "glue-core>=1.20.0",
+    "glue-core>=1.21.1",
```

Comment:
Also would have to uncomment the thingy from tox.ini after https://github.com/spacetelescope/jdaviz/pull/3401 is merged.